clients/js/packages/chromadb/package.json

```diff
@@ -24,24 +24,28 @@
     "dist"
   ],
   "scripts": {
-    "prebuild": "rimraf dist",
-    "build": "NODE_OPTIONS=--enable-source-maps tsup",
+    "prebuild": "rimraf dist && cd ../../../../rust/js_bindings && npm install && npm run build && cd -",
+    "build": "CHROMADB_VERSION=$npm_package_version NODE_OPTIONS=--enable-source-maps tsup",
+    "postbuild": "mkdir -p ./node_modules/chroma-rust-bindings && cp ../../../../rust/js_bindings/*.node ./node_modules/chroma-rust-bindings/",
     "watch": "NODE_OPTIONS=--enable-source-maps tsup --watch",
     "test": "jest --runInBand"
   },
   "dependencies": {
     "@google/generative-ai": "^0.1.1",
     "@xenova/transformers": "^2.17.2",
     "chromadb-default-embed": "^2.14.0",
+    "chromadb-rust-bindings": "file:../../../../rust/js_bindings/",
```

Comment:
this won't work when people are installing it

(i.e. try doing `npm build && npm pack`, then try installing the built `.tgz` in a new npm project somewhere)